src/content/blog/astro-570.mdx

```diff
@@ -0,0 +1,235 @@
+---
+title: 'Astro 5.7'
+description: 'Astro 5.7 has a basketload of treats, including stable Sessions and SVG components and a new Experimental Fonts API.'
+homepageLink:
+  title: 'Astro 5.7'
+  subtitle: 'Available now!'
+publishDate: '2025-04-15'
+authors:
+  - matt
+  - florian
+  - ema
+  - nate
+coverImage: '/src/content/blog/_images/astro-570/blog-post-5-7.webp'
+socialImage: '/src/content/blog/_images/astro-570/og-astro-5-7.webp'
+lang: 'en'
+---
+
+import BlogContentImage from '/src/components/BlogContentImage.astro';
+import Note from '~/components/Note.astro';
+import Mention from '/src/components/Mention.astro';
+
+**Astro 5.7 has a basketload of treats, including stable Sessions and SVG components and a new Experimental Fonts API.**
+
+🐣 Full of the joys of northern-hemisphere spring, Astro 5.7 is here with goodies galore:
+
+- [**Experimental Fonts API**](#experimental-fonts-api)
+- [**Sessions API**](#sessions-api)
+- [**SVG Components**](#svg-components)
+- [**Config Imports**](#config-imports)
+
+To upgrade an existing project, use the automated `@astrojs/upgrade` CLI tool. Alternatively, upgrade manually by running the upgrade command for your package manager:
+
+```sh
+# Recommended:
+npx @astrojs/upgrade
+
+# Manual:
+npm install astro@latest
+pnpm upgrade astro --latest
+yarn upgrade astro --latest
+```
+
+## Experimental Fonts API
+
+...
+
+## Sessions API
+
+The Astro Sessions API is now stable and ready for production!
+
+Sessions let you securely store data tied to a specific user as they navigate between pages on your site. Unlike cookies, sessions are stored on the server, so you can store larger amounts of data without worrying about size limits or security issues. They are useful for storing things like user data, shopping carts, and form state, and they work without any client-side JavaScript:
```

Comment:
This is a great paragraph, and I've updated docs with some of this exact wording! 🚀